### PR TITLE
Fix CSS animation examples

### PR DESCRIPTION
**Pull request by wbamberg**
The examples for our CSS animation properties play on page load, so they are done by the time you see them.

This PR makes them play on hover instead, so you can see them when you are ready.

I've also given them more titley titles.

### CODE REVIEW

File: files/en-us/web/css/animation-delay/index.md [hunk 1]
```diff
@@ -91,7 +96,7 @@ animation-delay: unset;
 
 #### Result
```
Comment (teoli2003) [inline files/en-us/web/css/animation-delay/index.md:98]: Should we say that the animation will start on hover?

```suggestion

Hover over the rectangle to start the animation.

```

File: files/en-us/web/css/animation-duration/index.md [hunk 2]
```diff
@@ -87,7 +92,7 @@ animation-duration: unset;
 
 #### Result
```
Comment (teoli2003) [inline files/en-us/web/css/animation-duration/index.md:94]:
```suggestion

Hover over the rectangle to start the animation.

```

File: files/en-us/web/css/animation-iteration-count/index.md [hunk 3]
```diff
@@ -92,7 +97,7 @@ The **`animation-iteration-count`** property is specified as one or more comma-s
 
 #### Result
```
Comment (teoli2003) [inline files/en-us/web/css/animation-iteration-count/index.md:99]:
```suggestion

Hover over the rectangle to start the animation.

```

File: files/en-us/web/css/animation-name/index.md [hunk 4]
```diff
@@ -89,7 +94,7 @@ animation-name: unset;
 
 #### Result
```
Comment (teoli2003) [inline files/en-us/web/css/animation-name/index.md:96]:
```suggestion

Hover over the rectangle to start the animation.

```

File: files/en-us/web/css/animation-play-state/index.md [hunk 5]
```diff
@@ -87,7 +94,7 @@ animation-play-state: unset;
 
 #### Result
```
Comment (teoli2003) [inline files/en-us/web/css/animation-play-state/index.md:96]:
```suggestion

Hover over the rectangle to start the animation.

```

File: files/en-us/web/css/animation-timeline/index.md [hunk 6]
```diff
@@ -104,7 +104,7 @@ A scroll timeline named `squareTimeline` is declared and applied to the `#square
 
 #### Result
```
Comment (teoli2003) [inline files/en-us/web/css/animation-timeline/index.md:106]:
```suggestion

Hover over the rectangle to start the animation.

```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/css/animation-duration/index.md (hunk 2 above)
Comment (wbamberg) [reply files/en-us/web/css/animation-duration/index.md:94]: Yeah, I guess so. I thought it was obvious, but worth spelling out.

File: files/en-us/web/css/animation-play-state/index.md [hunk 7]
```diff
@@ -87,7 +94,9 @@ animation-play-state: unset;
...
 
-{{EmbedLiveSample("Examples","100%","250")}}
+Hover over the rectangle to start the animation.
```
Comment (wbamberg) [inline files/en-us/web/css/animation-play-state/index.md:97]:
```suggestion
Hover over the rectangle to play the animation.
```

(this one does play/pause)

File: files/en-us/web/css/animation-timeline/index.md (hunk 6 above)
Comment (wbamberg) [reply files/en-us/web/css/animation-timeline/index.md:106]: (this one is on scroll)

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**